From: Jan Engelhardt <jengelh@inai.de>
Date: 2019-08-26 14:13:15.685236019 +0200

varnish screwing itself by use of -Werror.

[   34s] vtc_haproxy.c: In function 'haproxy_start':
[   34s] vtc_haproxy.c:442:10: error: 'pid' may be used uninitialized in this function [-Werror=maybe-uninitialized]
[   34s]   442 |   h->pid = pid;
[   34s]       |          ^
[   34s] vtc_haproxy.c:413:8: note: 'pid' was declared here
[   34s]   413 |  pid_t pid;

---
 bin/varnishtest/vtc_haproxy.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: varnish-6.2.0/bin/varnishtest/vtc_haproxy.c
===================================================================
--- varnish-6.2.0.orig/bin/varnishtest/vtc_haproxy.c
+++ varnish-6.2.0/bin/varnishtest/vtc_haproxy.c
@@ -410,7 +410,7 @@ haproxy_wait_pidfile(struct haproxy *h)
 	char buf_err[1024] = {0};
 	int usleep_time = 1000;
 	double t0;
-	pid_t pid;
+	pid_t pid = 0;
 
 	vtc_log(h->vl, 3, "wait-pid-file");
 	for (t0 = VTIM_mono(); VTIM_mono() - t0 < 3;) {
